core/src/banking_stage/vote_worker.rs

```diff
@@ -201,6 +196,65 @@ impl VoteWorker {
             .fetch_add(consumed_buffered_packets_count, Ordering::Relaxed);
     }
 
+    // returns `true` if the end of slot is reached
+    pub fn process_packets(
```

Comment:
nit: can be private

Reply:
a223d3c4a85e4a2623f3248f1da3f22f4b76cb77